site/src/components/form.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Form
+description: The form related components in the Atlas Design System
+template: standard
+---
+
+# Form
+
+Forms contain form control elements such as the following:
+
+- [`.button`](./button.md)
+- [`.checkbox`](./checkbox.md)
+- [`.help`](./help.md)
+- [`.input`](./input.md)
+- [`.label`](./label.md)
+- [`.radio`](./radio.md)
+- [`.select`](./select.md)
+- [`.textarea`](./textarea.md)
+- custom form elements
+
+## Form Field
+
+The `.field` component is a container for form control elements to provide consistency.
+It can be used to hold `.control` containers, which holds the individual control elements.
+
+`.field-label` is used on the label. Add `field-label-sm`/`field-label-lg` makes label a little bigger or smaller than the default.
+
+```html
+<div class="field">
+	<label class="field-label field-label-sm label" for="sample-form-input-sm"> Input </label>
+	<div class="control">
```

Comment:
can we rename the control class to something `field-`? eg `field-body`? I think something like that would align better with the naming standards and disambiguate control from the [control mixin](https://github.com/microsoft/atlas-design/blob/main/css/src/components/form/input.scss#L20)

Reply:
How about `field-section`? I'm thinking a field can have multiple sections, with each section containing an element and its label. If a field can only hold a single element, then `field-body` would be appropriate as it would be weird to have multiple bodies.